spec.emu

```diff
@@ -47,7 +48,8 @@ contributors: Krzysztof Kotowicz, Mike Samuel, Luke Warlow
   _parameterStrings_ represents the strings that, when using one of the function constructors, will be concatenated together to build the parameters list.
   _bodyString_ represents the function body or the string passed to an `eval` call.
   <del>_direct_ signifies whether the evaluation is a direct eval.</del>
-  <ins>_compilationType_ is an enum that indicates what type of compilation this is,
+  <ins>_codeString_ represents the string that will be compiled,
+  _compilationType_ is an enum that indicates what type of compilation this is,
```

Comment:
```suggestion
  <ins>_codeString_ represents the string that will be compiled,</ins>
  <ins>_compilationType_ is an enum that indicates what type of compilation this is,</ins>
```

Ecmarkup might accept unclosed `<ins>`s, but given that ecmarkup isn't 100% HTML it's better to explicitly close them.

Reply:
It's closed at the end of the _bodyArg_ line. Is it better for them to be individually opened and closed?